### DISCUSSION TITLE
Feedback: ambigous end/cancel voting wording

### DISCUSSION THREAD

**Discussion by Schwehn42** [inovex/scrumlr.io#3055] [Proposals]
I've gotten some feedback that the wording for ending or cancelling a voting phase is quite similiar to each other. This ambiguity leads to the user having to stop and think about what does exactly what.  
Maybe we should find a wording that is more concise to keep apart?

**Comment by brandstetterm** [comment]
This discussion (and underlying problem) has been resolved by #3931 because the option to abort a voting has been removed entirely.
